pokemongo_bot/__init__.py

```diff
@@ -342,7 +342,7 @@ def _register_events(self):
         )
         self.event_manager.register_event(
             'pokemon_evolved',
-            parameters=('pokemon', 'iv', 'cp', 'ncp', 'dps', 'xp')
```

Comment:
It is just that there is so many attributes that we could display, it is hard to content everyone. Also moves and DPs are changing at evolve so it's kind of wrong info.
So I take another approach, which is sticking with the classical iv/cp and add the rewards XP,candies.